apps/communications/dialer/test/marionette/PO_test.js

```diff
@@ -0,0 +1,77 @@
+/* global require, marionette, setup, suite, test*/
+'use strict';
+
+var assert = require('chai').assert;
+
+var Dialer = require('./lib/dialer.js');
+//var CallLogView = require('./lib/views/call_log/views.js');
+var DialerOptionsView = require('./lib/views/dialer_options/views.js');
+
+marionette('Dialer > Keypad', function() {
+
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'focusmanager.testmode': true
+      }
+    },
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var dialerApp;
+  var actions;
+
+  setup(function() {
+    actions = client.loader.getActions();
+    dialerApp = Dialer.create(client);
+    dialerApp.launch();
+  
+   });
+
+  suite('Test Suite', function() {
+
+    //setup(function(){
+    //    dialerApp.launch();
+        //reflowHelper = new ReflowHelper(client);
+        //dialerApp.DialerOptions.goToCallLog();
+        //subject.DialerOptions.toContactList(); 
+    //});
+
+    test('Check the call log', function() {
+        
+        //var option = dialerApp.DialerOptions;
+        //client.waitFor(function() {
+        //return option.callLog.enabled();
+        //});
+
+        //this works
+        //option.callLog.tap();
+        //this also work and returns a callLogView
+        //option.goToCallLog();
+
+        var dialerOptionsView = new DialerOptionsView(client);
```

Comment:
The main entry point for the test should be dialer. Here a way to do it could be: `dialer.tabs.goToCallLog()`